helm/libero-reviewer/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: "postgresql"
+    version: "6.2.0"
```

Comment:
May want to relax this to 6.x as the version is locked in requirements.lock; but have to check the notation to do so

Reply:
/cc @erezmus you like how Composer works?

Reply:
so would you write as:
```
version: "~6"
```
?